{
  "Header": {
    "Original": "CVE-2018-12564 (An issue was discovered in Linaro LAVA before 2018.5.post1. Because of ...)",
    "Line": 290389,
    "ID": "CVE-2018-12564",
    "Description": "(An issue was discovered in Linaro LAVA before 2018.5.post1. Because of ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4234-1 DLA-1404-1}",
      "Line": 290390,
      "Type": "xref",
      "Bugs": [
        "DSA-4234-1",
        "DLA-1404-1"
      ]
    },
    {
      "Original": "- lava 2018.5.post1-1",
      "Line": 290391,
      "Type": "package",
      "Package": "lava",
      "Kind": "fixed",
      "Version": "2018.5.post1-1"
    },
    {
      "Original": "- lava-server \u003cremoved\u003e",
      "Line": 290392,
      "Type": "package",
      "Package": "lava-server",
      "Kind": "removed"
    },
    {
      "Original": "NOTE: https://git.linaro.org/lava/lava.git/commit/?id=95a9a77b144ced24d7425d6544ab03ca7f6c75d3",
      "Line": 290393,
      "Type": "NOTE",
      "Description": "https://git.linaro.org/lava/lava.git/commit/?id=95a9a77b144ced24d7425d6544ab03ca7f6c75d3"
    }
  ]
}